iocage_lib/ioc_exec.py

```diff
@@ -262,4 +262,7 @@ def exec_jail(self):
 class SilentExec(object):
     def __init__(self, *args, **kwargs):
         with IOCExec(*args, **kwargs) as silent:  # noqa
-            pass
+            self.output = list(silent)
+
+        self.stdout = ''.join([i[0] for i in self.output])
```

Comment:
We should honor `decode` here and return byte strings by default

Reply:
Actually that is being taken care of in IOCExec ?